message/validation/genesis/consensus_validation.go

```diff
@@ -203,6 +205,8 @@ func (mv *messageValidator) validateSignerBehaviorConsensus(
 ) error {
 	signerState := state.GetSignerState(signer)
 
+	// If signer state is nil, this is the first message for the signer and
+	// the next rules can't be checked.
```

Comment:
I'd reword this because those rules (that check rules against state) can be checked but there's no reason for that because the state is empty

Reply:
Oh okok. Thanks!!

Reply:
Fixed